src/Symfony/Component/Workflow/Tests/Validator/SinglePlaceWorkflowValidatorTest.php

```diff
@@ -9,15 +9,15 @@
 use Symfony\Component\Workflow\Validator\SinglePlaceWorkflowValidator;
 use Symfony\Component\Workflow\Workflow;
 
-class SinglePlaceWorkflowValidatorTest extends WorkflowTest
+class SinglePlaceWorkflowValidatorTest extends \PHPUnit_Framework_TestCase
 {
     /**
      * @expectedException \Symfony\Component\Workflow\Exception\InvalidDefinitionException
      * @expectedExceptionMessage The marking store of workflow "foo" can not store many places.
      */
     public function testSinglePlaceWorkflowValidatorAndComplexWorkflow()
     {
-        $definition = $this->createComplexWorkflow();
+        $definition = WorkflowTest::createComplexWorkflow();
```

Comment:
Maybe just create a minimal workflow here. I thought of this as well for #20492 but i forgot :))